modules/scheduler/gke-cluster/main.tf

```diff
@@ -92,6 +92,13 @@ resource "google_container_cluster" "gke_cluster" {
 
   deletion_protection = var.deletion_protection
 
+  dynamic "enable_k8s_beta_apis" {
+    for_each = var.enable_k8s_beta_apis != null ? [""] : []
```

Comment:
Is there a reason for using [""] for true condition?
If the intention is for the loop to run for each of the APIs part of the list of strings, we don't need this condition. A simple `for_each = var.enable_k8s_beta_apis` should suffice by making the default an empty list using `[]`.

Reply:
Let me do this